### PR TITLE
Add Russian language

### PR DESCRIPTION
**Pull request by sturivny**
Used Poedit Version 3.0

### CODE REVIEW

**Comment by sturivny** [comment]
@dcantrell As changed file too big, it should be checked in terms of syntax typos.
Like `\n`, `%s` should be in the place as well as in the English variant.

**Comment by dcantrell** [comment]
This is really cool!  Thank you for sending it.  Checking the file now and will add comments where there should be changes.

**Comment by sturivny** [comment]
Thanks.
Also, I'm not sure about the LINGUAS file, should I update it?
Do let me know if something should be updated.

File: po/ru.po [hunk 1]
```diff
@@ -0,0 +1,2471 @@
...
+msgid ""
+"The referenced shell script is invalid. Consider debugging it with the '-n' "
+"option on the shell to find and fix the problem."
+msgstr ""
+"Указанный shell скрипт недействителен. Рассмотрите возможность отладки с "
+"помощью опции '-n' в терминале, чтобы найти и исправить проблему."
+
+#: lib/results.h:129
+msgid "See annocheck(1) for more information."
+msgstr "См. Дополнительную информацию в annocheck (1)."
```
Comment (dcantrell) [inline po/ru.po:1858]: Space between the command and the man page section (1)

File: po/ru.po [hunk 2]
```diff
@@ -0,0 +1,2471 @@
...
+
+#: lib/results.h:139
+msgid ""
+"Unexpected capabilities were found on the indicated file.  Consult "
+"capabilities(7) and either adjust the files in the package or modify the "
+"capabilities whitelist file in the rpminspect vendor data package.  The "
+"security team may also be of help for this situation."
+msgstr ""
+"В указанном файле обнаружены непредвиденные функции. Обратитесь к функциям "
+"(7) и либо настройте файлы в пакете, либо измените файл белого списка "
```
Comment (dcantrell) [inline po/ru.po:1898]: This (7) I think goes with capabilities(7) in which case I would say the capabilities part should not be translated unless there's also a capabilities(7) man page.

File: po/ru.po [hunk 3]
```diff
@@ -0,0 +1,2471 @@
...
+
+#: lib/rmtree.c:62
+#, c-format
+msgid "%s is not a directory.\n"
+msgstr "%s - это не каталог.\n"
+
+#: lib/rpm.c:74
+#, c-format
+msgid "*** Fopen() failed for %s: %s\n"
+msgstr "*** Ошибка Fopen () для%s:%s\n"
```
Comment (dcantrell) [inline po/ru.po:1985]: Space between Fopen and ()

**Review by dcantrell — CHANGES_REQUESTED** [review]

**Comment by dcantrell** [comment]
Also, do not commit the .mo file.  That is generated when you build and install the software.

And yes, the LINGUAS file needs to be updated using the language code that would appear in /usr/share/locale.

File: po/ru.po [hunk 4]
```diff
@@ -0,0 +1,2471 @@
...
+msgstr "*** Невозможно прочитать профиль '%s' с %s\n"
+
+#: lib/inspect_addedfiles.c:96
+#, c-format
+msgid ""
+"Packages should not contain not files or directories starting with `%s` on "
+"%s: %s"
+msgstr ""
+"Пакеты не должны содержать не файлы или каталоги, начинающиеся с `%s` в %s: "
+"%s"
```
[the remark this answers is not in the document]
Comment (sturivny) [reply po/ru.po:235]: Updated

File: po/ru.po [hunk 5]
```diff
@@ -0,0 +1,2471 @@
...
+
+#: lib/inspect_addedfiles.c:107
+#, c-format
+msgid ""
+"Packages should not contain files or directories ending with `%s` on %s: %s"
+msgstr ""
+"Пакеты не должны содержать файлы или каталоги, заканчивающиеся на `%s` в %s: "
+"%s"
```
[the remark this answers is not in the document]
Comment (sturivny) [reply po/ru.po:243]: Done

File: po/ru.po [hunk 6]
```diff
@@ -0,0 +1,2471 @@
...
+"убедитесь, что объекты ELF в после сборки не потеряли -D_FORTIFY_SOURCE. "
+"Наконец, если есть список запрещенных библиотечных функций, убедитесь, что "
+"они не используются."
+
+#: lib/inspect.c:126
+msgid ""
+"Perform syntax and file reference checks on *.desktop files. Syntax errors "
+"and invalid file references are reported as errors."
+msgstr ""
+"Выполните проверку синтаксиса и ссылок на файлы в файлах * .desktop. "
```
[the remark this answers is not in the document]
Comment (sturivny) [reply po/ru.po:377]: Done

File: po/ru.po [hunk 7]
```diff
@@ -0,0 +1,2471 @@
...
+"changes with diff output are included in the results."
+msgstr ""
+"Отчет об измененных файлах от до сборки до сборки. Определенные изменения "
+"файла вызовут дополнительные предупреждения, если проблема важнее,  чем "
+"просто сообщение об изменениях (например, связанная с безопасностью).  Любые "
+"сжатые файлы gzip, bzip2 или xz будут сравниваться только со своим  несжатым "
+"содержимым, что позволит изменять используемый уровень сжатия. Файлы "
+"каталога сообщений (.mo) распаковываются и сравниваются с помощью diff (1). "
+"Публичные заголовочные файлы C и C ++ предварительно обрабатываются и "
+"сравниваются с использованием diff (1). Любые изменения с выводом diff "
```
[the remark this answers is not in the document]
Comment (sturivny) [reply po/ru.po:431]: Yep, you are right, It should be the same.

File: po/ru.po [hunk 8]
```diff
@@ -0,0 +1,2471 @@
...
+"Message catalog files (.mo) are unpacked and compared using diff(1).  Public "
+"C and C++ header files are preprocessed and compared using diff(1).  Any "
+"changes with diff output are included in the results."
+msgstr ""
+"Отчет об измененных файлах от до сборки до сборки. Определенные изменения "
+"файла вызовут дополнительные предупреждения, если проблема важнее,  чем "
+"просто сообщение об изменениях (например, связанная с безопасностью).  Любые "
+"сжатые файлы gzip, bzip2 или xz будут сравниваться только со своим  несжатым "
+"содержимым, что позволит изменять используемый уровень сжатия. Файлы "
+"каталога сообщений (.mo) распаковываются и сравниваются с помощью diff (1). "
```
[the remark this answers is not in the document]
Comment (sturivny) [reply po/ru.po:429]: Fixed

File: po/ru.po [hunk 9]
```diff
@@ -0,0 +1,2471 @@
...
+
+#: lib/inspect_metadata.c:114
+#, c-format
+msgid "Gained Package Vendor \"%s\" in %s"
+msgstr "Получил вендора пакета \\\"%s\\\" in %s"
+
+#: lib/inspect_metadata.c:116
+#, c-format
+msgid "Lost Package Vendor \"%s\" in %s"
+msgstr "Потерялся вендора пакета \\\"%s\\\" in %s"
```
[the remark this answers is not in the document]
Comment (sturivny) [reply po/ru.po:1251]: Agree, updated

File: po/ru.po [hunk 10]
```diff
@@ -0,0 +1,2471 @@
...
+
+#: lib/magic.c:50
+#, c-format
+msgid "*** Unable to initialize the magic library\n"
+msgstr "*** Невозможно инициализировать волшебную библиотеку\n"
+
+#: lib/magic.c:56
+#, c-format
+msgid "*** Unable to load the magic database: %s\n"
+msgstr "\"*** Невозможно загрузить волшебную базу данных: %s\n"
```
[the remark this answers is not in the document]
Comment (sturivny) [reply po/ru.po:1512]: Fixed

File: po/ru.po (hunk 1 above)
Comment (sturivny) [reply po/ru.po:1858]: Done

File: po/ru.po (hunk 2 above)
Comment (sturivny) [reply po/ru.po:1898]: Updated

File: po/ru.po (hunk 3 above)
Comment (sturivny) [reply po/ru.po:1985]: Done

File: po/ru.po [hunk 11]
```diff
@@ -0,0 +1,2471 @@
...
+#, c-format
+msgid ""
+"File %s is not a valid desktop file on %s; desktop-file-validate reports:"
+msgstr ""
+"Файл %s не валидный десктопный файл на %s; desktop-file-validate отчет:"
+
+#: lib/inspect_disttag.c:85
+#, c-format
+msgid "The %s file is missing the Release: tag."
+msgstr "%s файл отчутствует для релиза: тег."
```
[the remark this answers is not in the document]
Comment (sturivny) [reply po/ru.po:844]: Yes, you are absolutely right. Updated.

File: po/ru.po [hunk 12]
```diff
@@ -0,0 +1,2471 @@
...
+
+#: lib/inspect_disttag.c:90
+#, c-format
+msgid "The dist tag should be of the form '%%{?dist}' in the Release tag."
+msgstr "Тег dist должен иметь вид '%%{?dist}' в теге релиза."
```
[the remark this answers is not in the document]
Comment (sturivny) [reply po/ru.po:849]: ditto

**Comment by sturivny** [comment]
> Also, do not commit the .mo file. That is generated when you build and install the software.
> 
> And yes, the LINGUAS file needs to be updated using the language code that would appear in /usr/share/locale.

Updated

File: po/LINGUAS [hunk 13]
```diff
@@ -1,3 +1,4 @@
 # list of supported languages using locale codes
 # (i.e., the subdir names in /usr/share/locale)
 #en_US
+#ru_RU
```
Comment (dcantrell) [inline po/LINGUAS:4]: The ru_RU line needs to be uncommented.  I put "#en_US" as an example, but as a comment so it would not mess up the build.

**Review by dcantrell — CHANGES_REQUESTED** [review]

**Comment by dcantrell** [comment]
The po file changes look good.  Only thing is to modify the LINGUAS file and uncomment ru_RU.

**Review by dcantrell — CHANGES_REQUESTED** [review]
Looks good.  This is really cool.  Since you did this, I realized it had been a while since I updated the translation template.  I just pushed an updated template and it looks like most things are the same, but can you double check everything and make sure all the strings are there?

**Comment by sturivny** [comment]
Sure

**Comment by sturivny** [comment]
Updated with the latest version of the rpminspect.pot

**Review by dcantrell — APPROVED** [review]

**Pull request merged**